results-processor/processor.py

```diff
@@ -257,9 +257,16 @@ def update_status(self, run_id, stage, error=None, callback_url=None):
         payload = {'id': int(run_id), 'stage': stage}
         if error:
             payload['error'] = error
-        revision = self.report.run_info.get('revision')
-        if revision:
-            payload['full_revision_hash'] = revision
+        if self.report.run_info.get('revision'):
```

Comment:
There are a variety of renames here (revision --> full_revision_hash, product --> browser_name, etc). Are these commonly taken to be well known why they happen, or should they be documented at all?

Reply:
This is largely historical. And it turns out we never really document the mapping apart from in the source code. Filed #1716 .